permission_prompt/explariner_navigation.md

```diff
@@ -0,0 +1,89 @@
+# Private Network Access Permission to relax mixed content for non-fetch requests
+
+- **Author**: lyf@google.com
+- **Created**: 2023-11-24
+- **Last Updated**: 2024-01-26
+
+## A problem
+
+Currently, any public page can make a request as if they were the user's machine. This allows a malicious attacker to use a public web page to attack network endpoints on a user's private network. These endpoints (router, printer, IoT devices, ...) are often badly protected because they do not expect to receive requests from the web at large.
+
+To mitigate this threat, we have been working on the Private Network Access (PNA) spec, which prevents public pages from making requests to private network endpoints, unless the endpoint opts into receiving the request from the public origin. This model works if the origin can be validated, i.e. if the public page is served over HTTPS. To support PNA, we want to remove the ability for public HTTP pages to make requests on private network endpoints.
+
+This means that public pages served on HTTP that embed resources from the private network need to upgrade to HTTPS. However, there is currently no good solution for encryption on the private network, meaning that many of those subresources are served over HTTP and cannot upgrade to HTTPS. They cannot be embedded in an HTTPS page due to mixed contents checks.
+
+To solve this issue, we have introduced a permission prompt to relax mixed contents checks when fetching resources from the private network. However, it only covers subresources but not iframes. We want to extend this mechanism to navigation.
```

Comment:
I feel like we should elaborate more on the fourth paragraph on why the current solution only covers subresources but not iframe, i.e. because requests via `fetch` API can specify the `targetAddressSpace`.   The first three paragraphs are general permission prompt problems which can be merged to a shorter paragraph with a link to the permission prompt explainer.

Also, since you've already updated the title to cover "non-fetch" requests, maybe you should just put the "Furthermore" section here.